htdocs/contact/consumption.php

```diff
@@ -300,6 +300,7 @@
 	$sql.= " FROM "/*.MAIN_DB_PREFIX."societe as s, "*/.$tables_from;
 //	if ($type_element != 'fichinter') $sql.= ' LEFT JOIN '.MAIN_DB_PREFIX.'product as p ON d.fk_product = p.rowid ';
 	$sql.= $where;
+	$sql.= ' AND ec.fk_socpeople = '.$object->id;
```

Comment:
The table called "ec" was added in request with a INNER JOIN or a LEFT JOIN. This means you must not add condition in the global where but you have to put it after the "... ON ..." that is set just after the INNER or LEFT JOIN.

Reply:
@eldy 
it is a tab that is in the contact card, should not we filter with the contact ID globally? This tab does not allow the contact to see only the documents with which they are associated?

Reply:
@eldy 
ok i send a fix
maybe it would be good to mutualized SQL queries because many of them are duplicates